# Thomas Hoffmann, EMBL Heidelberg, structures-it@embl.de, 2023/11
# install command_line.py to bin. (to be renamed later by postinstall) 
diff -ru relion-blush-3148869de884fc01cff6c51a047b1413fc0b62cb/setup.py relion-blush-3148869de884fc01cff6c51a047b1413fc0b62cb_add_command_line_to_bin/setup.py
--- relion-blush-3148869de884fc01cff6c51a047b1413fc0b62cb/setup.py	2023-11-05 14:39:04.000000000 +0100
+++ relion-blush-3148869de884fc01cff6c51a047b1413fc0b62cb_add_command_line_to_bin/setup.py	2023-11-14 16:27:58.081283230 +0100
@@ -35,6 +35,7 @@
     packages=find_packages(),
     install_requires=install_requires,
     python_requires=">=3.5",
+    scripts = ["relion_blush/command_line.py"]
 )
 
 # Download, install and load model
